FlowCrypt/Common/Imap.swift

```diff
@@ -22,7 +22,8 @@ final class Imap {
 
     struct EmptyError: Error {}
     private typealias ReqKind = MCOIMAPMessagesRequestKind
-    private var lastErr = [String: MCOErrorCode]();
+    private typealias Err = MCOErrorCode
+    private var lastErr = [String: Err]();
```

Comment:
minor, but complier better. and remove `;` 
`private var lastErr: [String: Err] = []`

Reply:
Ah, ok!

Reply:
It seems like it wants `[:]`